lib/ExponentialBackoff.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var ExponentialBackoff = function(minimum_backoff, exponent, starting_backoff) {
```

Comment:
please use doc blocks for functions (at a minimum short description and `@param` lines with types)